corehq/apps/api/accounting.py

```diff
@@ -172,7 +172,7 @@ class InvoiceResource(ModelResource):
     applied_credit = fields.DecimalField('applied_credit')
 
     class Meta(AccountingResourceMeta):
-        queryset = Invoice.objects.all().order_by('pk')
+        queryset = Invoice.api_objects.all().order_by('pk')
```

Comment:
@czue 
After my changes in ```Invoice``` we will have two managers. 
The first one 
https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/accounting/models.py#L1675 is working in HQ app whereas this manager filters records by flag ```is_hidden_to_ops```
The second is needed in API because we need all records there, That's why I added default manager from django.db.models.

Is that make sense?